docs/user/resources/05-metricpipeline.md

```diff
@@ -179,24 +179,24 @@ The status of the MetricPipeline is determined by the condition types `GatewayHe
 | AgentHealthy           | True             | AgentReady                  | Metric agent DaemonSet is ready                                                                                                                                                                                                          |
-| AgentHealthy           | True             | AgentReady                  | Pods are being started/updated                                                                                                                                                                                                           |
+| AgentHealthy           | True             | RolloutInProgress           | Pods are being started/updated                                                                                                                                                                                                           |
 | AgentHealthy           | False            | AgentNotReady               | No Pods deployed                                                                                                                                                                                                                         |
 | AgentHealthy           | False            | AgentNotReady               | DaemonSet is not yet created                                                                                                                                                                                                             |
 | AgentHealthy           | False            | AgentNotReady               | Failed to get DaemonSet                                                                                                                                                                                                                  |
-| AgentHealthy           | False            | AgentNotReady               | Pod is in the pending state as container: `container name` is not running due to: `reason`                                                                                                                                                   |
-| AgentHealthy           | False            | AgentNotReady               | Pod is in the failed state due to: `reason`                                                                                                                                                                                                  |
+| AgentHealthy           | False            | AgentNotReady               | Pod is in the pending state as container: `container name` is not running due to: `reason`                                                                                                                                               |
```

Comment:
```suggestion
| AgentHealthy           | False            | AgentNotReady               | Pod is in the pending state because container: `container name` is not running due to: `reason`                                                                                                                                               |
```